demo/lib/vm.bash

```diff
@@ -79,14 +85,15 @@ vm-check-env() {
 vm-check-binary-cri-resmgr() {
     # Check running cri-resmgr version, print warning if it is not
     # the latest local build.
+    # shellcheck disable=SC2016
     if [ -f "$BIN_DIR/cri-resmgr" ] && [ "$(vm-command-q 'md5sum < /proc/$(pidof cri-resmgr)/exe')" != "$(md5sum < "$BIN_DIR/cri-resmgr")" ]; then
         echo "WARNING:"
         echo "WARNING: Running cri-resmgr binary is different from"
         echo "WARNING: $BIN_DIR/cri-resmgr"
         echo "WARNING: Consider restarting with \"reinstall_cri_resmgr=1\" or"
         echo "WARNING: run.sh> uninstall cri-resmgr; install cri-resmgr; launch cri-resmgr"
         echo "WARNING:"
-        sleep ${warning_delay}
+        sleep "${warning_delay-0}"
```

Comment:
nitpick: personally I've tried to consistently use `:-` everywhere in bash scripts instead of this. IMO, it's even more readable.

Reply:
Thanks, changed `warning_delay` defaults to `:-` syntax. I'll move to the same practice for code base consistency - and readability. :)